go/shuffle/read.go

```diff
@@ -595,8 +617,8 @@ func pickHRW(h uint32, from []shuffleMember, start, stop int) int {
 }
 
 // readChannelCapacity is sized so that sendReadResult will overflow and
-// cancel the read after ~8 seconds of no progress (1 << (14 - 1) millis).
-var readChannelCapacity = 15
+// cancel the read after ~65 seconds of no progress (1<<15 + 1<<14 + 1<<13 ... millis).
```

Comment:
Does this math still add up after changing the channel capacity to 17?

Reply:
Yep, it's 1 << (len(ch)-1), and we never let ch fill to capacity